raiden/tests/unit/transfer/test_utils.py

```diff
@@ -0,0 +1,17 @@
+from eth_utils import decode_hex
+from raiden.transfer.state import EMPTY_MERKLE_ROOT
+from raiden.transfer.utils import hash_balance_data
+
+
+def test_hash_balance_data():
+    test_table = {
+        (0, 0, b''): decode_hex(
+            '0x0000000000000000000000000000000000000000000000000000000000000000',
+        ),
+        (1, 5, EMPTY_MERKLE_ROOT): decode_hex(
+            '0xc6b26a4554afa01fb3409b3bd6e7605a1c1af45b7e644282c6ebf34eddb6f893',
+        ),
+    }
+
+    for values, result in test_table.items():
```

Comment:
nitpick: you could use pytest for this: https://github.com/ethereum/eth-utils/blob/master/tests/hexadecimal-utils/test_is_0x_prefixed.py#L8

Reply:
side note though: this feature is really slow, don't use it with a large list (for this is fine)